docs/vagrant.md

```diff
@@ -23,51 +30,15 @@ To ssh in the cluster, run following:
 vagrant ssh
 ```
 
-The password for vagrant user is **vagrant'**
-
-## Setting Up YARN/Hadoop ##
-
-To setup YARN/Hadoop inside VM, run following YARN setup shell files:
-
-1 Run the first YARN setup shell command from the vagrant directory to create a user hduser in group hadoop. Be sure to remember the password that you provide for this user.
-```
-cd /vagrant
```

Comment:
Looks like the vagrant setup is a lot simpler now. Would you mind updating the wiki page as well?

https://cwiki.apache.org/confluence/display/MYRIAD/Installing+using+Vagrant